AnnService/inc/Core/Common/NeighborhoodGraph.h

```diff
@@ -380,9 +381,6 @@ namespace SPTAG
                     auto t2 = std::chrono::high_resolution_clock::now();
                     LOG(Helper::LogLevel::LL_Info, "Refine RNG time (s): %lld Graph Acc: %f\n", std::chrono::duration_cast<std::chrono::seconds>(t2 - t1).count(), GraphAccuracyEstimation(index, 100, idmap));
                 }
-                else {
-                    LOG(Helper::LogLevel::LL_Info, "Graph Acc: %f\n", GraphAccuracyEstimation(index, 100, idmap));
```

Comment:
Keep this part to test the quality of GPU graph.

Reply:
Added back in with latest push